network/address.go

```diff
@@ -67,7 +68,81 @@ func (a Address) NetworkAddress() string {
 		return ""
```

Comment:
Can you change the comments of the function, now that it performs a DNS Lookup ?